src/app/kvm/views/LXDClusterDetails/LXDClusterHostSettings/LXDClusterHostSettings.tsx

```diff
@@ -4,25 +4,33 @@ import { Spinner, Strip } from "@canonical/react-components";
 import { useSelector } from "react-redux";
 import { useNavigate } from "react-router-dom-v5-compat";
 
-import { useWindowTitle } from "app/base/hooks";
+import ModelNotFound from "app/base/components/ModelNotFound";
+import { useGetURLId, useWindowTitle } from "app/base/hooks";
 import KVMConfigurationCard from "app/kvm/components/KVMConfigurationCard";
 import LXDHostToolbar from "app/kvm/components/LXDHostToolbar";
 import SettingsBackLink from "app/kvm/components/SettingsBackLink";
 import { useActivePod, useKVMDetailsRedirect } from "app/kvm/hooks";
+import kvmURLs from "app/kvm/urls";
 import podSelectors from "app/store/pod/selectors";
-import type { Pod } from "app/store/pod/types";
+import { PodMeta } from "app/store/pod/types";
 import { isPodDetails } from "app/store/pod/utils";
 import type { RootState } from "app/store/root/types";
 import vmClusterSelectors from "app/store/vmcluster/selectors";
 import type { VMCluster } from "app/store/vmcluster/types";
+import { isId } from "app/utils";
 
 type Props = {
   clusterId: VMCluster["id"];
-  hostId: Pod["id"];
 };
 
-const LXDClusterHostSettings = ({ clusterId, hostId }: Props): JSX.Element => {
+export enum Label {
+  Loading = "Loading LXD host",
+  Title = "LXD cluster host settings",
+}
+
+const LXDClusterHostSettings = ({ clusterId }: Props): JSX.Element => {
   const navigate = useNavigate();
+  const hostId = useGetURLId(PodMeta.PK, "hostId");
```

Comment:
Previously these ids were passed in from the routes file, but RRv6 needs `useParams` (which is used inside `useGetURLId`) to be called inside a route for the params to be captured.